tailwind.config.js

```diff
@@ -4,12 +4,33 @@ module.exports = {
   content: ['./app/**/*.{ts,tsx,jsx,js}'],
   theme: {
     extend: {
+      borderRadius: {
+        16: '4rem',
+      },
       colors: {
+        grey: {
+          DEFAULT: '#D6D3D1',
+          med: '#78716C',
+          dark: '#44403C',
+        },
         resolutionBlue: '#041484',
         razzmatazz: '#E30464',
         feijoa: '#96D882',
         fern: '#6DB470',
       },
+      fontSize: {
```

Comment:
we dont need to be updating tailwind config in this `PR`, also i think we have the option to already add font size as 14px, 18px by default from tailwind